src/components/list/README.md

```diff
@@ -105,6 +105,33 @@ Output:
 
 <img src="https://material.angularjs.org/material2_assets/list/dense-list.png">
 
+### Lists with multiple sections
+
+You can add a subheader to a list by wrapping your subheader text in `<md-subheader>` tags, and you can 
+add dividers with `<md-divider>` tags.
+
+```html
+<md-list>
+   <md-subheader>Folders</md-subheader>
```

Comment:
Oh, now that you mention it, we did talk about it. I can update it to be an attribute selector.